src/vs/workbench/contrib/chat/browser/media/chat.css

```diff
@@ -1182,6 +1182,11 @@ have to be updated for changes to the rules above, or to support more deeply nes
 	border-color: var(--vscode-focusBorder);
 }
 
+.chat-attached-context-attachment .monaco-icon-label > .monaco-icon-label-container > .monaco-icon-suffix-container > .label-suffix {
+	color: var(--vscode-peekViewTitleDescription-foreground);
```

Comment:
Is the peek view token the right thing to use here, since that might normally be on a different background color than the peek view?

Reply:
i figured it might be okay since this is only in chat pills, but @kkbrooks i'm not sure if you've looked into where else it's used and if it passes the accessibility stuff? 

otherwise, could just be worth making a new token like we talked about